.rubocop.yml

```diff
@@ -1,31 +1,12 @@
 AllCops:
   Exclude:
+    - Rakefile
+    - Berksfile
+    - metadata.rb
     - vendor/**/*
-    - Guardfile
-
-AlignParameters:
-  Enabled: false
-ClassLength:
-  Enabled: false
-CyclomaticComplexity:
-  Enabled: false
+    - spec/**/*
```

Comment:
If possible we'd like to keep the rubocop configs on the test/spec dirs.  It's nice that they're consistent and readable as well.